source/jormungandr/jormungandr/interfaces/v1/Journeys.py

```diff
@@ -291,20 +291,18 @@ def wrapper(*args, **kwargs):
                 return objects
             if "journeys" not in objects[0]:
                 return objects
-            if "region" in kwargs:
-                del kwargs["region"]
             if "uri" in kwargs:
-                kwargs["from"] = kwargs["uri"].split("/")[-1]
                 del kwargs["uri"]
             if "lon" in kwargs and "lat" in kwargs:
-                if not "from" in kwargs:
-                    kwargs["from"] = kwargs["lon"] + ';' + kwargs["lat"]
                 del kwargs["lon"]
                 del kwargs["lat"]
             for journey in objects[0]['journeys']:
-                if "sections" not in journey:
+                if "sections" not in journey:#this mean it's an isochrone...
                     kwargs["datetime"] = journey["requested_date_time"]
                     kwargs["to"] = journey["to"]["id"]
+                    kwargs["from"] = journey["from"]["id"]
+                    if 'datetime_represents' in request.args:
+                        kwargs['datetime_represents'] = request.args['datetime_represents']
```

Comment:
I think it would be easier to just deepcopy the request.args dict and just add a `to` or a `from` if there is none